profiles/base.yaml

```diff
@@ -183,7 +183,7 @@ cambrinth_items:
 loot_additions:
 loot_subtractions:
 loot_specials:
-
```

Comment:
You need to add `stop_hunt_on_box_limit`

Reply:
no. removed.